### ISSUE TITLE
failed swagger json import - "The URI does not contain a valid specification"

### ISSUE THREAD

**Issue by scollovati** [getinsomnia/insomnia#2810]
**Describe the bug**
Hi, Insomnia is failing in importing this swagger.json: file https://ultra-us-prod-csa.bbcollab.cloud/swagger.json

RangeError: Maximum call stack size exceeded

**To Reproduce**
1. Click on workspace name > "import/export"
2. Import from URL https://ultra-us-prod-csa.bbcollab.cloud/swagger.json
3. See error "The URI does not contain a valid specification"
[](url)
RangeError: Maximum call stack size exceeded
    at object (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:545:13)
    at generateParameterExample (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Collovati\AppData\Local\insomnia\app-2020.4.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)

**Desktop (please complete the following information):**
 - OS: Windows10
 - App Version 2020.4.2

**Comment by DevYukine** [comment]
Same issue here, using Swashbuckle sadly can't be imported

Stacktrace:
```
RangeError: Maximum call stack size exceeded
    at Array.values (<anonymous>)
    at object (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:550:43)
    at generateParameterExample (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
    at generateParameterExample (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:592:12)
    at object (C:\Users\Raphael\AppData\Local\insomnia\app-2020.5.2\resources\app.asar\node_modules\insomnia-importers\src\importers\openapi3.js:551:35)
```

**Comment by DevYukine** [comment]
This is fixed as of latest release 2021.1.0

**Comment by dimitropoulos** [comment]
![Screenshot_20210325_104229](https://user-images.githubusercontent.com/15232461/112491568-d1b5f980-8d56-11eb-99ab-819bb6da2d49.png)

I was also able to import this document.  please let us know if this issue is not now resolved for you and we can reopen this issue and investigate.
